episodes/introduction.md

```diff
@@ -81,6 +81,13 @@ accessibility purposes'}`
 
 ![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.'}
 
+::::::::::::::::::::::::::::::::::::: callout
+
+Callout sections can highlight information.
```

Comment:
```suggestion
Callout sections can highlight information.

They are sometimes used to emphasise particularly important points
but are also used in some lessons to present "asides": 
content that is not central to the narrative of the lesson,
e.g. by providing the answer to a commonly-asked question.
```

I suggest expanding this explanation, to reflect the dual use of callouts I have observed in the community.